tests/browser/heda_settings_accounts_and_contacts_edit_mode.robot

```diff
@@ -0,0 +1,64 @@
+*** Settings ***
+
+Resource        tests/HEDA.robot
+Suite Setup     Open Test Browser
+Suite Teardown  Delete Records and Close Browser
+
+*** Test Cases ***
+
+Go To Heda Home
+    Go To Heda Home
+    Select Frame			//iframe[contains(@id, "vfFrameId")]
+    Click Link				//a[contains(text(),'Accounts and Contacts')]
+    Wait Until Element Is visible	//a[contains(text(),'Accounts and Contacts')]
+
+Validate Edit Mode For Accounts and Contacts, Settings
+    
+    
+
+    #Go into Edit mode 
+    Click Element                   //div[@class='slds-button-group']//span[contains(text(), 'Edit')]
+    Wait Until Element Is visible   //div[@class='slds-button-group']//span[contains(text(), 'Save')] 
+
+
+    Wait Until Element Is visible   //div[@class='slds-tabs--default']//span[contains(text(),'Academic Program')]
+    Click Element                   //div[@class='slds-tabs--default']//span[contains(text(),'Academic Program')]
+
+    Wait Until Element Is visible   //div[@class='slds-tabs--default']//span[contains(text(),'Administrative')]
+    Click Element                   //div[@class='slds-tabs--default']//span[contains(text(),'Administrative')]
+
+    Wait Until Element Is visible   //div[@class='slds-tabs--default']//span[contains(text(),'Business Organization')]
+    Click Element                   //div[@class='slds-tabs--default']//span[contains(text(),'Business Organization')]
+
+    Wait Until Element Is visible   //div[@class='slds-tabs--default']//span[contains(text(),'Educational Institution')]
+    Click Element                   //div[@class='slds-tabs--default']//span[contains(text(),'Educational Institution')]
+
+    Wait Until Element Is visible   //div[@class='slds-tabs--default']//span[contains(text(),'Household Account')]
+    Click Element                   //div[@class='slds-tabs--default']//span[contains(text(),'Household Account')]
+
+    Wait Until Element Is visible   //div[@class='slds-tabs--default']//span[contains(text(),'Sports Organization')]
+    Click Element                   //div[@class='slds-tabs--default']//span[contains(text(),'Sports Organization')]
+
+    Wait Until Element Is visible   //div[@class='slds-tabs--default']//span[contains(text(),'University Department')]
+    Click Element                   //div[@class='slds-tabs--default']//span[contains(text(),'University Department')]
+
+
+
+
+    Click Element                   //div[@class='slds-form-element__control']//span[contains(text(),'Academic Program')]
```

Comment:
Same here, this could be a generic keyword defined in python with a locator accepting an argument

Reply:
Also, this test feels like it should be split up into multiple test cases so it's easier to figure out what the failure was.